docs/content/config/index.md

```diff
@@ -31,7 +37,10 @@ We also provide an `.env.example` file containing a minimal configuration in the
 
 ### Why should I want to run my renderer on a different (sub-)domain?
 
-If the renderer is provided by another domain, it's way harder to manipulate HedgeDoc or steal credentials from the rendered note content, because renderer and editor are more isolated. This increases the security of the software and greatly mitigates [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting). However, you can run HedgeDoc without this extra security, but we recommend using it if possible.
+If the renderer is provided by another domain, it's way harder to manipulate HedgeDoc or
+steal credentials from the rendered note content, because renderer and editor are more isolated.
+This increases the security of the software and greatly mitigates [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+However, you can run HedgeDoc without this extra security, but we recommend using it if possible.
 
```

Comment:
Add that this requires own configuration adjustments to the reverse proxy (so that it serves the frontend at two domains)